deployment/articles/03-installing-a-search-engine/03-solr/03-installing-solr-solrcloud.markdown

```diff
@@ -0,0 +1,127 @@
+---
+header-id: high-availability-with-solrcloud
+---
+
+# Installing Solr: High Availability with SolrCloud
+
+[TOC levels=1-4]
+
+Use SolrCloud if you need a cluster of Solr servers. Note that to use SolrCloud
+in production, you should set up an 
+[external ZooKeeper ensemble](https://cwiki.apache.org/confluence/display/solr/Setting+Up+an+External+ZooKeeper+Ensemble).
+[ZooKeeper](http://zookeeper.apache.org/) is a centralized coordination service
+for managing distributed systems like your SolrCloud cluster.
+
+The steps included here should be considered the bare minimum of what must be
+done to configure SolrCloud with @product@. For example, these instructions cover
+configuring SolrCloud on a single machine, whereas a production environment
+would feature multiple physical or virtual machines. These instructions also
+assume you've followed the earlier section on *Installing and Configuring Solr
+7*. Refer to the [SolrCloud guide for more information](https://cwiki.apache.org/confluence/display/solr/SolrCloud).
+
+1. Stop the Solr server if it's running.
+
+2. Navigate to the `Solr_Home/configsets` folder and create a folder called 
+
+        liferay_configs
+
+3. Copy the `conf` folder from `Solr_Home/liferay` to the `liferay_configs`
+   folder you just created.
+
+    The `configset/liferay_configs` folder contains the SolrCloud
+    @product@ collection configuration and is uploaded to ZooKeeper. By copying
+    the `conf` folder from the `liferay` server configured earlier, you're using
+    the `schema.xml` and `solrconfig.xml` files provided with the Liferay Solr
+    Adapter.
+
+4. Next launch an interactive SolrCloud session to configure your SolrCloud
+   cluster. Use this command:
+
+        ./bin/solr -e cloud
+
+5. Complete the setup wizard. These steps demonstrate creating a two-node
+   cluster:
+
+    -  Enter `2` for the number of nodes.
+    -  Specify ports `8983` and `7574` (the defaults). Both nodes are
+       started with the start commands printed in the log:
```

Comment:
@rbohl 

We are missing the command for the 2nd node:

```bash
Starting up Solr on port 8983 using command:
"bin/solr" start -cloud -p 8983 -s "example/cloud/node1/solr"

Starting up Solr on port 7574 using command:
"bin/solr" start -cloud -p 7574 -s "example/cloud/node2/solr" -z localhost:9983
```

Reply:
It was intentional. I just saw no point in pasting in both of them.